behaviors/wow_retail/ExampleBehavior/ExampleBehavior.lua

```diff
@@ -1,7 +1,9 @@
+local DemonhunterHavoc = require('behaviors.wow_retail.ExampleBehavior.demonhunter.havoc')

```

Comment:
Example must only be one file to keep it compact, easy to understand easy to maintain.

Don't make the example be a full rotation, at most one spell cast if you include logic and it must be a spell available on level 1 for that class so that the example can be tested on a fresh character.

Reply:
This is really the Pallas behaviour.  Once you're happy with the structure, then I'll add a real example behaviour

Reply:
As soon as this works how we want, i change this to Pallas retail and populate it, the example behavior will have ALL classes (im guessing in both games) with your suggested level 1 skill as well as "initial" specs.